fed/mosaic-application/src/main/java/org/eclipse/mosaic/fed/application/app/api/os/modules/Sensible.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2024 Fraunhofer FOKUS and others. All rights reserved.
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contact: mosaic@fokus.fraunhofer.de
+ */
+
+package org.eclipse.mosaic.fed.application.app.api.os.modules;
+
+import org.eclipse.mosaic.fed.application.app.api.sensor.BasicSensorModule;
+import org.eclipse.mosaic.fed.application.app.api.sensor.LidarSensorModule;
+import org.eclipse.mosaic.lib.enums.SensorType;
+
+/**
+ * Interface to mark an {@link org.eclipse.mosaic.fed.application.app.api.os.OperatingSystem} as
+ * an owner of a {@link BasicSensorModule} and {@link LidarSensorModule} to sense data from the surrounding environment.
+ */
+public interface Sensible {
```

Comment:
I understand the intention behind introducing this marker interface "Sensible" and separate it from the "Perceptive"-interface. However, I believe this distinction may not be necessary. Both interfaces seem to overlap in their core purpose of handling information related to the environment.

Reply:
I'm totally with you! I moved the methods to the `Perceptive` interface and removed the `Sensible` interface.